llm-servers/deepsparse/Containerfile

```diff
@@ -0,0 +1,8 @@
+FROM registry.access.redhat.com/ubi9/python-39
```

Comment:
wondering if we can bump this to 3.11 as in the vllm case

Reply:
I'll give it a shot

Reply:
awesome! thanks!